packages/itwinui-css/src/button/split-button.scss

```diff
@@ -31,7 +30,7 @@
   }
 
   :where(.iui-button:last-child) {
-    --_iui-button-padding-inline: #{$iui-xs};
+    --_iui-button-padding-inline: var(--iui-size-2xs);
```

Comment:
```suggestion
    --_iui-button-padding-inline: var(--iui-size-2xs);
    
    min-width: unset;
```
We need to remove the min-width on the 2nd button within the split button.

How it looks currently:
![Screen Shot 2022-08-12 at 10 00 34 AM](https://user-images.githubusercontent.com/849817/184369291-8f2a55af-4c30-4e18-ade3-3ec055816410.png)

How it should look:
![Screen Shot 2022-08-12 at 10 01 36 AM](https://user-images.githubusercontent.com/849817/184369479-a0bd5bf8-f94e-4f06-bae7-20aa6e3ca0f8.png)